### PR TITLE
FEATURE/MINOR: parsers: add additional global directives

### PR DESCRIPTION
**Pull request by anitgandhi**

### CODE REVIEW

File: section-parsers.go [hunk 1]
```diff
@@ -176,6 +176,7 @@ func getGlobalParser() *Parsers {
 		&parsers.LogSendHostName{},
 		&parsers.LuaLoad{},
 		&parsers.ConfigSnippet{},
+		&simple.Time{Name: "hard-stop-after"},
```
Comment (oktalz) [inline section-parsers.go:186]: Snippet need to be the last one, please move this in front of parsers.Log{} parser

**Comment by oktalz** [comment]
it looks good (minor revision is needed).
I need to re-think about `simple.StringSlice` a bit, 
and primarily how this is ( or can be) used in config-snippet (it could be OK)

File: section-parsers.go (hunk 1 above)
Comment (anitgandhi) [reply section-parsers.go]: fixed 👍

**Comment by oktalz** [comment]
sorry for the delay, merging

**Pull request merged**